AwsCryptographicMaterialProviders/runtimes/java/src/main/smithy-generated/software/amazon/cryptography/materialproviders/MaterialProviders.java

```diff
@@ -50,9 +50,10 @@ protected MaterialProviders(BuilderImpl builder) {
     MaterialProvidersConfig input = builder.MaterialProvidersConfig();
     software.amazon.cryptography.materialproviders.internaldafny.types.MaterialProvidersConfig dafnyValue =
       ToDafny.MaterialProvidersConfig(input);
-    Result<MaterialProvidersClient, Error> result = __default.MaterialProviders(
-      dafnyValue
-    );
+    //BEGIN MANUAL FIX
+    Result<IAwsCryptographicMaterialProvidersClient, Error> result =
+      __default.MaterialProviders(dafnyValue);
+    //END MANUAL FIX
```

Comment:
@robin-aws can you cut a GHI against Smithy-Dafny for this?
Or a fix?

Reply:
Weird, when I run `make polymorph_java` locally I see the correct output already.

I'm adding CI to regenerate this code with a smithy-dafny submodule and diff it against what's checked in, so that will confirm whether the tip of smithy-dafny has this bug or if this didn't actually need to be flagged as a manual edit after all.